src/config/index.md

```diff
@@ -1,3 +1,13 @@
 # Configuration
```

Comment:
Perhaps make it clear that there are pages under this section? I think section contents would be too much, but at least make it explicit that the configuration section isn't only the shell part?

Reply:
Yeah, i was thinking the same thing, but couldn't immediately come up with wording i didn't find awkward. (At one point i had: "This section and its subsections provide information about configuring your Void
system.")

Reply:
That doesn't sound too bad, and it's definitely better than nothing at all.

Reply:
Oh okay - well, i've put it back in. :-)